core/src/test/scala/com/rackspace/com/papi/components/checker/GivenAWadlWithMetaTransforms.scala

```diff
@@ -0,0 +1,279 @@
+  val targetCustomService = "/custom/metadata/service:foo"
+  val putContent = <meta/>
+
+  val targetsStandard = List(
+    "",
+    "billing:",
+    "service:"
+  )
+
+  val targetsCustom = List(
+    "",
+    "customer:role:",
+    "service:"
+  )
+
+  // http://stackoverflow.com/questions/13109720/how-to-create-all-possible-combinations-from-the-elements-of-a-list
+  val rolesSet = Set("admin", "billing:role", "service:role", "superAdmin", "customer:role", "another_role")
+  val rolesList = rolesSet.subsets.map(_.toList).toList
+
+  // Standard GET's are allowed regardless of roles.
+  targetsStandard.foreach { target =>
+    rolesList.foreach { roles =>
+      it should behave like accessIsAllowed(validator, methodGet, s"/standard/metadata/${target}foo", roles, description)
+    }
+    it should behave like accessIsAllowedWhenNoXRoles(validator, methodGet, s"/standard/metadata/${target}foo", description)
+  }
+
+//  // Standard PUT's and DELETE's are allowed or forbidden based on roles.
+// @TODO: Original double loop with PUT's and DELETE's for Standard targets.
+//  List(methodPut, methodDelete).foreach { method =>
+//    targetsStandard.foreach { target =>
+//      rolesList.foreach { roles =>
+//        // IF the list of roles contains the target
+//        // OR the list of roles contains admin,
+//        // THEN access should be allowed;
+//        // ELSE access should be forbidden.
+//        if (targetInRoles(target, roles, List("admin"))) {
+//          it should behave like accessIsAllowedWithContent(validator, method, s"/standard/metadata/${target}foo", roles, putContent, description)
+//        } else {
+//          it should behave like accessIsForbidden(validator, method, s"/standard/metadata/${target}foo", roles, description)
+//        }
+//      }
+//      it should behave like accessIsForbiddenWhenNoXRoles(validator, method, s"/standard/metadata/${target}foo", description)
+//    }
+//  }
+
+  targetsStandard.foreach { target =>
+    rolesList.foreach { roles =>
+      // IF the list of roles contains the target
+      // OR the list of roles contains admin,
+      // THEN access should be allowed;
+      // ELSE access should be forbidden.
+      if (targetInRoles(target, roles, List("admin"))) {
+        // @TODO: Huh???
+        // @TODO: What should the content be?
+        // @TODO: Do we need the new accessIsAllowedWithContent() method?
+        // @TODO: Should we just keep the new badRequestWithStockContent() method since we know it made it all the way through?
+        //it should behave like accessIsAllowed(validator, methodPut, s"/standard/metadata/${target}foo", roles, description)
+        //it should behave like accessIsAllowedWithContent(validator, methodPut, s"/standard/metadata/${target}foo", roles, putContent, description)
+        it should behave like badRequestWithStockContent(validator, methodPut, s"/standard/metadata/${target}foo", roles, description)
```

Comment:
That's the correct behavior.  Check this out here are some examples of what metadata requests look like. https://github.com/rackerlabs/metadata-guide/tree/master/src/docbkx/api_samples

If you build that project you'll get docs which should give you more context on how metadata is supposed to work.